docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -0,0 +1,554 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d689eeca",
+   "metadata": {},
+   "source": [
+    "# Create and Use Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "548cbdc4",
+   "metadata": {},
+   "source": [
+    "## About Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19ceffc1",
+   "metadata": {},
+   "source": [
+    "Functions are the basic building blocks of MLRun - they will be used to run jobs, deploy models, create pipelines, and more. You can read more about MLRun Functions [here](https://docs.mlrun.org/en/latest/runtimes/functions.html)."
```

Comment:
Functions are the basic building blocks of MLRun. They are used to run jobs, deploy models, create pipelines, and more. You can read more about MLRun Functions [here](https://docs.mlrun.org/en/latest/runtimes/functions.html).